src/Document_Item.php

```diff
@@ -457,44 +428,55 @@ public static function showForDocument(Document $doc)
                     }
 
                     $link     = $itemtype::getFormURLWithID($data['id']);
-                    $name = "<a href='$link'>$linkname $linkname_extra</a>";
-
-                    echo "<tr class='tab_bg_1'>";
-
-                    if ($canedit) {
-                        echo "<td width='10'>";
-                        Html::showMassiveActionCheckBox(__CLASS__, $data["linkid"]);
-                        echo "</td>";
+                    $name = "<a href='$link'>" . htmlspecialchars($linkname) . ' ' . htmlspecialchars($linkname_extra) . "</a>";
```

Comment:
Links may also contains special chars (e.g. `&`) and we do not know what kind of data plugins can provide in links.

```suggestion
                    $name = '<a href="' . htmlspecialchars($link) . '">' . htmlspecialchars($linkname) . ' ' . htmlspecialchars($linkname_extra) . "</a>";
```